### ISSUE TITLE
Order of pools in GSLB Domain (WideIP)

### ISSUE THREAD

**Issue by MMarco1977** [F5Networks/f5-appsvcs-extension#482]
Environment

    Application Services Version: 3.28.0
    BIG-IP Version: 15.1.3-0.0.11

Summary:
Although in the documentation it is stated that the pools will be inserted with the sequence indicated in the command, the result is random (the pools appear with the value Order random)

Reference Documentation:
_Specifies the pools that this domain uses for load balancing. The system evaluates the pools in the order in which they are listed, until it finds a matching pool_

### Steps To Reproduce
1. Submit the following declaration:
```json
[
  { 
    "op": "add",
    "path": "/Common/Shared/ACME-FELI",
    "value": { 
                "class": "GSLB_Server",
                "serverType": "generic-host",
                "monitors": [
                        
                    ],
                "dataCenter": { 
                    "use": "marco"
                },
                "devices":[
                    {
                        "label": "ACME-FELI",
                        "address": "1.2.9.8"
                    } 
                ],
                "virtualServers": [
                    {
                        "name": "ACME-FELI_VS",
                        "address": "1.2.9.8",
                        "port": 0
                    }
                ]
            }
    },
    { 
    "op": "add",
    "path": "/Gsv/Application/pool_FELI",
    "value": { 
                "class": "GSLB_Pool",
                "resourceRecordType": "A", 
                "lbModeFallback": "none",
                "lbModeAlternate": "none",
                "verifyMemberEnabled": false,
                "members": [
                    {
                        "server": {
                            "use": "/Common/Shared/ACME-FELI" 
                        },
                        "virtualServer": "ACME-FELI_VS",
                        "enabled": true
                    }
                ]
            }
    },
    { 
    "op": "add",
    "path": "/Gsv/Application/nicola.info.it/pools",
    "value": [
               {
                "use": "pool_FELI"
                },
                {
                "use": "pool_GIANM"
                },
                {
                "use": "pool_nomatch"
                }
            ]
    }
]
```

Expected Behavior:
I expect the order in the declaration to be respected, instead as you can see from the screen, the order is random

<img width="912" alt="Screenshot 2021-07-15 at 13 01 52" src="https://user-images.githubusercontent.com/85392708/125777800-ab8bdc62-ee43-4e2e-9fc6-be8ef7673284.png">

**Comment by dstokesf5** [comment]
Thank you for reporting this. I have confirmed that the order property is missing when AS3 generates wideip objects. I have added this to our internal product backlog as AUTOTOOL-2635.

**Comment by dstokesf5** [comment]
This was resolved with the 3.31.0 release of AS3.
